packages/sdk/src/sdk-blockchains/ethereum/bid.ts

```diff
@@ -138,12 +158,56 @@ export class EthereumBid {
 			multiple: collection.type === "ERC1155",
 			maxAmount: item.supply,
 			baseFee: await this.sdk.order.getBaseOrderFee(),
+			getConvertableValue: this.getConvertableValue,
+			convert: this.convertCurrency,
 			submit,
 		}
 	}
 
+	async getConvertableValue(
+		assetType: AssetType, value: BigNumberValue
+	): Promise<{ type: "insufficient" | "convertable", currency: AssetType; value: BigNumberValue } | undefined > {
+		if (!this.wallet) {
+			throw new Error("Wallet is undefined")
+		}
+		if (assetType["@type"] === "ETH") return undefined
```

Comment:
instead of checking that not belong to subset you should check what belongs to subset

like 
```
if ([if ERC20 or TEZOS_FT] && [exists in convertable currencies map]) {
 ... do calculations
} else {
 return undefined
}
```